test/doctool/test-apilinks.mjs

```diff
@@ -20,14 +18,28 @@ fs.readdirSync(apilinks).forEach((fixture) => {
 
   const expectedContent = fs.readFileSync(`${input}on`, 'utf8');
   const outputPath = tmpdir.resolve(`${fixture}on`);
+
+  fs.mkdirSync(outputPath);
+
   execFileSync(
-    process.execPath,
-    [script, outputPath, input],
+    'npx',
+    [
+      '--prefix',
+      docToolingDir,
+      'doc-kit',
+      'generate',
+      '-t',
+      'api-links',
+      '-i',
+      input,
+      '-o',
+      outputPath,
+    ],
```

Comment:
```suggestion
    process.execPath,
    [
      path.resolve(docToolingDir, 'node_modules', '.bin', 'doc-kit'),
      'generate',
      '-t',
      'api-links',
      '-i',
      input,
      '-o',
      outputPath,
    ],
```

This should also avoid using npx, it also assumes that there must be a `npx` on the path which is not necessarily true in the context of the test.